lib/Doctrine/DBAL/Schema/OracleSchemaManager.php

```diff
@@ -318,6 +318,9 @@ public function createDatabase($database = null)
      */
     public function dropAutoincrement($table)
     {
+        if ($table instanceof Table) {
```

Comment:
[`dropTable`](https://github.com/doctrine/dbal/blob/de6c0bd243d058eb9f086144a6fed9d435737c78/lib/Doctrine/DBAL/Schema/AbstractSchemaManager.php#L347) states that the given argument must be a string: callers should respect that